src/store/Account.js

```diff
@@ -248,6 +257,61 @@ const actions = {
       .catch(err => {
         commit(types.TRANSFER_ASSET_FAILURE, err)
       })
+  },
+
+  createAccount ({ commit }, { accountName, domainId }) {
+
+    let { publicKey, privateKey } = cryptoHelper.generateKeyPair()
+
+    if(!_.isEmpty(accountName.trim())){
+      return new Promise((resolve, reject) => {
+        return commands.createAccount(
+          newPreCommandServiceOptions(),
+          {
+            accountName: accountName,
+            domainId: domainId,
+            publicKey: publicKey
+          }
+        )
+        .then(() => {
+          resolve(privateKey)
+        })
+        .catch(err => { reject(err) })
+      })
+      .catch(err => {
+        commit(types.SIGNUP_FAILURE, err)
+      })
+    }
+  },
+
+  setAccountDetails ({ state }, { accountId = state.accountId, accountInfo }) {
+    const setAccountDetailArray = _.flatMap(accountInfo, function(value, key) {
+      return commands.setAccountDetail(
+        newPreCommandServiceOptions(), 
+        {
+          accountId: accountId,
+          key: key,
+          value: value
+        }
+      )
+      .catch(err => {
+        throw err
+      })
+    });
+
+    return Promise.all(setAccountDetailArray)
+  },
+
+  initialAssetQuantityForUser ({ state }, { accountId }) {
```

Comment:
It is function, but called like variable. Functions should be called like action with verb.